### PR TITLE
Email encoder/check bot before decode

### PR DESCRIPTION
**Pull request by safronik**

### CODE REVIEW

File: js/src/apbct-public--2--public.js [hunk 1]
```diff
@@ -102,7 +102,7 @@ function checkEmail(e) {
 	}
 }
 
-function ctSetPixelImg(pixelUrl) {
+function tSetPixelImg(pixelUrl) {
```
Comment (ArtemAnoshin) [inline js/src/apbct-public--2--public.js:105]: typo?

File: js/src/apbct-public--2--public.js [hunk 2]
```diff
@@ -411,6 +406,8 @@ function getJavascriptClientData() {
 	// Parse JSON properties to prevent double JSON encoding
 	resultDataJson = decodeJSONEncodedProperties(resultDataJson);
 
+	console.log(resultDataJson);
```
Comment (ArtemAnoshin) [inline js/src/apbct-public--2--public.js:409]: need to remove?

Comment (Glomberg) [reply js/src/apbct-public--2--public.js]: https://github.com/CleanTalk/wordpress-antispam/pull/164/commits/61b56ccfa4c4f1c855dae4f46fbcc02c19e480bb

File: js/src/apbct-public--2--public.js (hunk 1 above)
Comment (Glomberg) [reply js/src/apbct-public--2--public.js]: https://github.com/CleanTalk/wordpress-antispam/pull/164/commits/61b56ccfa4c4f1c855dae4f46fbcc02c19e480bb

**Review by Glomberg — APPROVED** [review]

**Pull request merged**